pytest_watch/watcher.py

```diff
@@ -92,8 +94,8 @@ def run(self, summary=None):
             print(STYLE_NORMAL + msg + Fore.RESET + Style.NORMAL)
         if self.beforerun:
             os.system(self.beforerun)
-        exit_code = subprocess.call(['py.test'] + self.args,
-                                    shell=(sys.platform == 'win32'))
+        exit_code = pytest.main(self.args, plugins=[ReportCollectorPlugin()])
```

Comment:
Ok.

In that case, can we have a external plugin set some value in pytest.config which would be passed on to the executable by ptw?

This way, ptw would have to not know about any plugins, just the value at `sessionexit`.